core/templates/pages/exploration-editor-page/services/change-list.service.ts

```diff
@@ -92,7 +92,13 @@ export class ChangeListService implements OnInit {
     private loaderService: LoaderService,
     private loggerService: LoggerService,
     private internetConnectivityService: InternetConnectivityService,
-  ) {
+  ) {}
+
+  ngOnInit(): void {
+    this.loaderService.onLoadingMessageChange.subscribe(
+      (message: string) => this.loadingMessage = message
+    );
+
     this.internetConnectivityService.onInternetStateChange.subscribe(
```

Comment:
Currently, this logic is in constructor which makes it hard to test the parity checks.
In order to make it testable I've copied the code into `ngOnInit`